docs/kyma/04-02-local-installation.md

```diff
@@ -19,88 +20,79 @@ Virtualization:
-      ```
-      </details>
-    </div>
-
-8. By default, the Kyma installation is a background process, which allows you to perform other tasks in the terminal window. Nevertheless, you can track the progress of the installation by running the `is-installed.sh` script. It is designed to give you clear information about the Kyma installation. Run it at any point to get the current installation status, or to find out whether the installation is successful.
-
-  ```
-  ./scripts/is-installed.sh
-  ```
-  >**TIP:** If the script indicates that the installation failed, try to install Kyma again by re-running the `run.sh` script. If the installation fails in a reproducible manner, don't hesitate to create a [GitHub](https://github.com/kyma-project/kyma/issues) issue in the project or reach out to the ["installation" Slack channel](https://kyma-community.slack.com/messages/CD2HJ0E78) to get direct support from the community.
-
-9. After the installation is completed, you can access the Console UI. Go to [this](https://console.kyma.local) address and select **Login with Email**. Use the **admin@kyma.cx** email address and the password you set using the `--password` flag.
-
-10. At this point, Kyma is ready for you to explore. See what you can achieve using the Console UI or check out one of the [available examples](https://github.com/kyma-project/examples).
+Follow these instructions to install Kyma from a release or from sources:
+<div tabs>
+  <details>
+  <summary>
+  From a release
+  </summary>
+    
+  1. Provision a Kubernetes cluster on Minikube. Run:
+    
+     ```bash
+     kyma provision minikube
+     ```
+     > **NOTE:** The `provision` command uses default Minikube VM driver installed for your operating system. For a list of supported VM drivers see [this document](https://kubernetes.io/docs/setup/minikube/#quickstart).
+  
+  2. Install Kyma on Minikube:
+     ```bash
+     kyma install 
+     ```
+ >**NOTE** If you want to install a specific release version, go to the [GitHub releases page](https://github.com/kyma-project/kyma/releases) to find out more about available releases. Use the release version as a parameter when calling ` kyma install --release {KYMA_RELEASE}`.
+  3. After the installation is completed, you can access the Console UI. Go to [this](https://console.kyma.local) address and select **Login with Email**. Use the **admin@kyma.cx** email address and the password printed in the terminal after installation finished.
+  
+  4. At this point, Kyma is ready for you to explore. See what you can achieve using the Console UI or check out one of the [available examples](https://github.com/kyma-project/examples).
+   
+  </details>
+  <details>
+  <summary>
+  From sources
+  </summary>
+    
+  1. Open a terminal window and navigate to a space in which you want to store local Kyma sources.
+    
+  2. Clone the `Kyma` repository using either HTTPS or SSH. Run:
```

Comment:
Now that this step is INSIDE a div element, I'd drop the two-option approach. It makes the document unnecessarily longer. Decide which one is more convenient for a newcomer (HTTP or SSH) and use that approach.

Reply:
I agree, lets go with HTTPS option only